app/views/layouts/unifiedLayout.scala.html

```diff
@@ -143,6 +143,7 @@
         bodyEndBlock = None,
         headerBlock = Some(siteHeader),
         footerBlock = Some(footerLinks()),
-        scriptsBlock = Some(scripts())
+        scriptsBlock = Some(scripts()),
+        assetPath = Some(uk.gov.hmrc.hmrcfrontend.controllers.routes.Assets.at("govuk").url)
```

Comment:
This will need to be added in the if logic as well or it won't work on the home page (where main class is defined)